src/DependencyInjection/StofDoctrineExtensionsExtension.php

```diff
@@ -149,6 +151,8 @@ public function load(array $configs, ContainerBuilder $container)
 
     /**
      * @internal
+     *
+     * @return void
```

Comment:
As this method is internal, you can add it as an actual return type without BC impact